hydra/test_utils/test_utils.py

```diff
@@ -283,19 +295,28 @@ def experiment(cfg):
             f.write("# @package _global_\n")
             OmegaConf.save(task_config, f)
         config_name = "config_name='config'"
-    output_file = str(tmpdir / "output.txt")
+
+    script_output_file = str(tmpdir / "output.txt")
+
+    # You can have our test script write to an arbitrary file
+    # if so, you are in charge of copying the arbitrary file to the validation dir.
+    validation_file = str(tmpdir / "output.txt")
```

Comment:
this is not really arbitrary. it's output.txt in the job output directory. this is 100% controlled by this function.

Reply:
removed, this variable was not needed anymore.